### DISCUSSION TITLE
Dịch core instance

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#71] [Terminology] [answered]
Từ này được dùng trong [DBSCAN](https://en.wikipedia.org/wiki/DBSCAN), mình thấy @rootonchair đang để là `mẫu chủ chốt`.

Mọi người thấy sao? @mlbvn/handson-ml2

**Comment by rootonchair** [comment, marked as answer]
@mlbvn/handson-ml2 chốt nhé
